kind/controlplane/create.go

```diff
@@ -55,6 +55,11 @@ func CreateKindCluster(clusterName string) error {
 		return errors.Wrap(err, "couldn't create control plane")
 	}
 
+	// Remove taint from the management cluster
+	if err := cp.Command("kubectl", "--kubeconfig=/etc/kubernetes/admin.conf", "taint", "nodes", "--all", "node-role.kubernetes.io/master-").Run(); err != nil {
```

Comment:
That would be a huge refactor of the manager and pretty out of scope for this PR. I'd LOVE to use client-go to do this. It would clean up the docker file and all sorts of things. I'll open an issue for this one.